java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -196,7 +201,7 @@ public ProgramState apply(ProgramState input) {
   }
 
   private void handleBlockExit(ExplodedGraph.ProgramPoint programPosition) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this method 6, down to the maximum allowed 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1142)